packages/location-icon/src/LocationIcon.story.js

```diff
@@ -17,8 +13,14 @@ storiesOf("LocationIcon", module)
       text: "A simple component used to show the from or to location icon"
     }
   })
-  .add("From LocationIcon", () => <LocationIcon type="from" size={25} />)
-  .add("To LocationIcon", () => <LocationIcon type="to" size={25} />)
-  .add("To LocationIcon test", () => (
-    <StyledLocationIcon type="to" size={25} />
-  ));
+  .add("'From' LocationIcon", () => <LocationIcon type="from" size={25} />)
+  .add("'To' LocationIcon", () => <LocationIcon type="to" size={25} />)
+  .add("Generic Place LocationIcon", () => (
+    <LocationIcon type="intermediate-place-1" size={25} />
+  ))
+  .add("Custom style for 'To' LocationIcon", () => {
+    const StyledLocationIcon = styled(LocationIcon)`
```

Comment:
Normally you'd follow [this advice](https://styled-components.com/docs/basics#define-styled-components-outside-of-the-render-method) but it's probably fine for rendering one story component.